lib/calendar_date_select/form_helpers.rb

```diff
@@ -144,13 +144,8 @@ def calendar_date_select(object, method, options={})
         end
       end
 
-    tag = ActionView::Helpers::InstanceTag.new_with_backwards_compatibility(object, method, self, options.delete(:object))
-    calendar_date_select_output(
-      tag.to_input_field_tag( (javascript_options[:hidden] || javascript_options[:embedded]) ? "hidden" : "text", options),
-      image,
-      options,
-      javascript_options
-    )
+    element = element_renderer_class(javascript_options).new(object, method, self, options)
```

Comment:
Looking at the first line that was removed `tag = ... options.delete(:object))`.

Should the new first line, `element = ...` also remove `:object` from `options` parameter?